content/chapters/software-stack/lab/media/syscall-explained.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2023-02-26T10:04:05.978Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36" etag="Fg2EAA14vL3nbjUvsp2d" version="20.8.23" type="device"><diagram name="Page-1" id="ZXZmJir3GP8aIHGH0DWt">7Vpdb9owFP01SNsDKI5DCI+BlrbaJlXqpm57mUxiEq8mzhxTyH797MT5Ih2lAgp046HEJzeJfc7x9Y1LB47nqyuO4vAT8zHtmIa/6sCLjmkOLFv+VUCaAxZwciDgxM8hUAF35DfWoKHRBfFx0ggUjFFB4ibosSjCnmhgiHO2bIbNGG0+NUYBbgF3HqJt9J74IsxRxxxU+DUmQVg8GdjD/MwcFcF6JEmIfLasQfCyA8ecMZEfzVdjTBV3BS/5dZO/nC07xnEktrng98/onrokGH+/vro1vofB8OZLt+jcI6ILPWLdW5EWFHC2iHys7mJ04GgZEoHvYuSps0upucRCMaeyBeRhIjh7wGNGGZdIxCIZNpqxSGhpgaXbRUjHhEb2kXh7SHqUj5gLvKpBeohXmM2x4KkM0Wf7+gptN2DrAS4r8SxDY2FNuBJE2jBBeeuKU3mgaX0BxcMWodiXFtNNxkXIAhYhelmhoyblVcxHxmJN9E8sRKpJRQvBmjJIAnn6VV3fM/tF+5u+X9a4WDVaqW419auLs7OICVtwD28yo57fiAdYbIjTc0/RuNESHFMkyGNzJu9dXwCPK3BD3V7/Dehr71vf7FKXc5TWAmJGIpHU7nyrgCqTgCIj6FRi2mv59Zn4PjDWvJX3oHJaOZQdzGcdObu8ueQCT8J8Zf1TmM/YbD64Of4w5jNbtcOXBHOJ5OWBaVNJ9GjKG/60fy1UxZMp3E0yyV0ZAKx4lelbnJdHgfp245gST1LNopbVn6lFUBLnJeGMrJTdRzNCac1VtoGGYLDBbdv7dQ+VCxisSWi2SxfnicrFOVThAkCL8NfNLYOzSS7wPCuX/lEFBp2zqVy21dc6icUDOi+rXNbjrdeoXOAui8eUcR/zrpeL7Gb34e+63Tr+/t9eTsxTW06cYySbYyYNa8uksffXnZ1kslrz8gPmkdrU22VbaG2qzAwEDXQqU6XcJErX8mFtqphPbRrZh5ordkuEa8T9pRzkXmUAUwSweaoyWEeXYdCS4S5NBJ5LbIwolUuw4d7e7FUSDD2nJPjkJIH9LSWBB3spab90tvhPQhSrQzLPNvfrbKuhy+WfupQEkcSEWkdK9COaYnrLEpKVB/BiyoRgcxlA1YkR8h6CTNt6Jss+MiR7mFuUCMZT9YLuz0UohPrvhauYMCeeH8Ee8Vg0I9I1vOfJJ5oTHwkkvxQubTaZydTfRZ7qV9LNQLWvPOkrOiafiffw4xPiD5IdpxdHwWEXvhfUIOaafUDbPvBV3dOe0f+wewZDReeYsgT/uJHoqdkHOlbTPk+UsK9rH+e/fc7HPqbVb9in3GA7mn2G/+1zPvZZe3+Gh3OPbFY/A8j3V6rfUsDLPw==</diagram></mxfile>
```

Comment:
Do not include .drawio files, we can import svg files in drawio and obtain the same result.